emerge/tests/parsers/test_parsing_mixin.py

```diff
@@ -63,7 +63,8 @@ def test_resolve_relative_dependency_path(self):
             scanned_by="parser_name",
             scanned_language=LanguageType.JAVASCRIPT,
             scanned_tokens="token1 token2",
-            preprocessed_source=""
+            source="",
```

Comment:
Unrelated to the original issue, but this was failing when running `python3 run_tests.py`